samples/snippets/README.md

```diff
@@ -0,0 +1,38 @@
+# google-cloud-compute library samples
+
+These samples demonstrate usage of the google-cloud-compute library to interact
+with the Google Compute Engine API.
+
+## Running the quickstart script
+
+### Before you begin
+
+1. If you haven't already, set up a Python Development Environment by following the [python setup guide](https://cloud.google.com/python/setup) and 
+[create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+1. Create a service account with the 'Editor' permissions by following these 
```

Comment:
I would recommend using `gcloud auth application-default login` instead.  It's more secure if you don't have to download a JSON key.   It's also easier.   You can skip the next 3 steps with this method.

Reply:
OK. Good point. Done.